src/corebluetooth.cs

```diff
@@ -665,6 +673,9 @@ interface CBUUID : NSCopying {
 		[Field ("CBUUIDCharacteristicAggregateFormatString")]
 		NSString CharacteristicAggregateFormatString { get; }
 
+		[Field ("CBUUIDCharacteristicValidRangeString")]
+		NSString CharacteristicValidRangeString { get; }
```

Comment:
There isn't a `StrongDictionary` for the other fields which I found surprising. Guess we could add one.